tsconfig.json

```diff
@@ -16,7 +16,7 @@
     "inlineSourceMap": true,
     "inlineSources": true,
     "experimentalDecorators": true,
-    "strictPropertyInitialization":false
+    "strictPropertyInitialization": false
   },
-  "exclude": ["cdk.out"]
+  "exclude": ["cdk.out/**/*", "node_modules/**/*"]
```

Comment:
Was this causing weird behaviour when depending on the package directly from `git`?

Reply:
```
npm ERR! command /Users/cyber/.nvm/versions/node/v12.16.3/bin/node /Users/cyber/.nvm/versions/node/v12.16.3/lib/node_modules/npm/bin/npm-cli.js install --only=dev --prod --ignore-prepublish --no-progress --no-save --cache=/Users/cyber/.npm/_cacache --prefer-offline=false --prefer-online=false --offline=false --before=
npm ERR! > serverless-aws-cdk@0.5.0 install
npm ERR! > npm run build
npm ERR!
npm ERR!
npm ERR! > serverless-aws-cdk@0.5.0 build
npm ERR! > tsc
npm ERR!
npm ERR! cdk/deployFunction.ts(6,21): error TS2307: Cannot find module 'fs' or its corresponding type declarations.
npm ERR! cdk/serverlessStack.ts(4,23): error TS2307: Cannot find module 'path' or its corresponding type declarations.
npm ERR! cdk/serverlessStack.ts(87,25): error TS2580: Cannot find name 'require'. Do you need to install type definitions for node? Try `npm i @types/node`.
npm ERR! cdk/toolkit.ts(14,5): error TS2580: Cannot find name 'process'. Do you need to install type definitions for node? Try `npm i @types/node`.
npm ERR! cdk/toolkitStack.ts(5,32): error TS2307: Cannot find module './environment' or its corresponding type declarations.
npm ERR! node_modules/aws-cdk/lib/api/util/cloudformation/stack-activity-monitor.d.ts(1,23): error TS2688: Cannot find type definition file for 'node'.
npm ERR! node_modules/aws-cdk/lib/api/util/cloudformation/stack-activity-monitor.d.ts(132,22): error TS2503: Cannot find namespace 'NodeJS'.
npm ERR! node_modules/aws-cdk/lib/api/util/cloudformation/stack-activity-monitor.d.ts(170,32): error TS2503: Cannot find namespace 'NodeJS'.
npm ERR! node_modules/aws-cdk/lib/cdk-toolkit.d.ts(1,23): error TS2688: Cannot find type definition file for 'node'.
npm ERR! node_modules/aws-cdk/lib/cdk-toolkit.d.ts(136,14): error TS2503: Cannot find namespace 'NodeJS'.
npm ERR! node_modules/aws-cdk/node_modules/@aws-cdk/cloudformation-diff/lib/format.d.ts(1,23): error TS2688: Cannot find type definition file for 'node'.
npm ERR! node_modules/aws-cdk/node_modules/@aws-cdk/cloudformation-diff/lib/format.d.ts(3,39): error TS2503: Cannot find namespace 'NodeJS'.
...


npm ERR! provider/awsCdkProvider.ts(5,23): error TS2307: Cannot find module 'path' or its corresponding type declarations.
npm ERR! provider/awsCdkProvider.ts(44,24): error TS2584: Cannot find name 'console'. Do you need to change your target library? Try changing the `lib` compiler option to include 'dom'.
npm ERR! provider/awsCdkProvider.ts(51,9): error TS2580: Cannot find name 'process'. Do you need to install type definitions for node? Try `npm i @types/node`.
npm ERR! provider/awsCdkProvider.ts(52,7): error TS2584: Cannot find name 'console'. Do you need to change your target library? Try changing the `lib` compiler option to include 'dom'.
npm ERR! provider/awsCdkProvider.ts(53,7): error TS2584: Cannot find name 'console'. Do you need to change your target library? Try changing the `lib` compiler option to include 'dom'.
npm ERR! provider/awsCdkProvider.ts(55,7): error TS2584: Cannot find name 'console'. Do you need to change your target library? Try changing the `lib` compiler option to include 'dom'.
npm ERR! provider/awsCdkProvider.ts(56,7): error TS2584: Cannot find name 'console'. Do you need to change your target library? Try changing the `lib` compiler option to include 'dom'.
npm ERR! provider/awsCdkProvider.ts(58,5): error TS2584: Cannot find name 'console'. Do you need to change your target library? Try changing the `lib` compiler option to include 'dom'.
npm ERR! provider/awsCdkProvider.ts(59,5): error TS2584: Cannot find name 'console'. Do you need to change your target library? Try changing the `lib` compiler option to include 'dom'.
```

Reply:
:+1: Cool, makes sense